### ISSUE TITLE
Testing question

### ISSUE THREAD

**Issue by WANGLEKV** [W-zx-Y/DANNet#11]
During the test, the generated results need to be uploaded to the codalab website. The folders to be uploaded are confidence, labeltraining and labeltraining_ Invalid, in which the pictures in the "confidence" folder are written in your testing code. What pictures do you need to put in the "labeltraining_ids" and "labeltraining ids_invalid" folders?  Can this part of the code be made public？Thank you !

**Comment by WANGLEKV** [comment]
There are limited opportunities to submit tests. How do you choose from many models you have trained?

**Comment by W-zx-Y** [comment]
Hi @WANGLEKV,
We put the generated semantic segmentation maps into the "labeltraining_ids", which are used for calculating the mIoU. Because we don't need the UIoU results, we upload the black images (all zeros) for "confidence" and copy all images of "labeltraining_ids" into "labeltraining ids_invalid". 
For our experiments, we chose the model using the validation set and then submitted the prediction of the test set to the codalab website.

**Comment by WANGLEKV** [comment]
> Hi @WANGLEKV, We put the generated semantic segmentation maps into the "labeltraining_ids", which are used for calculating the mIoU. Because we don't need the UIoU results, we upload the black images (all zeros) for "confidence" and copy all images of "labeltraining_ids" into "labeltraining ids_invalid". For our experiments, we chose the model using the validation set and then submitted the prediction of the test set to the codalab website.

**Comment by WANGLEKV** [comment]
thank you !!

**Comment by Urbenyy** [comment]
> 在测试过程中，生成的结果需要上传到 codalab 网站。要上传的文件夹是 confidence、labeltraining 和 labeltraining_ Invalid，其中 “confidence” 文件夹中的图片写在测试代码中。“labeltraining_ids”和“labeltraining ids_invalid”文件夹需要放什么图片？这部分代码可以公开吗？谢谢！

您好，冒昧打扰您，请问你解决这个问题了吗，我最近一直上传都是有问题的结果，按照作者写的步骤上传上去仍然有错误，Missing prediction for ground truth file GOPR0364_frame_000001_gt_labelTrainIds.png
======= UIoU=ERROR =======
Missing prediction for ground truth file GOPR0364_frame_000001_gt_labelTrainIds.png
======= Avg_UIoU=ERROR =======
Missing prediction for ground truth file GOPR0364_frame_000001_gt_labelTrainIds.png
======= IoU=ERROR =======
请问你是怎么解决的呢
